tools/java-source-utils/src/test/java/com/microsoft/android/JavadocXmlGeneratorTest.java

```diff
@@ -53,48 +60,50 @@ public void testWritePackages_demo() throws ParserConfigurationException, Transf
 		final   JavadocXmlGenerator     generator   = new JavadocXmlGenerator(new PrintStream(bytes));
 		final   JniPackagesInfo         packages    = JniPackagesInfoTest.createDemoInfo();
 
-		final   String                  expected    = 
-			"<?xml version=\"1.0\" encoding=\"UTF-8\" standalone=\"no\"?>\n" +
-			"<api api-source=\"java-source-utils\">\n" +
-			"  <package jni-name=\"\" name=\"\">\n" +
-			"    <class jni-signature=\"LA;\" name=\"A\">\n" +
-			"      <javadoc><![CDATA[jni-sig=LA;]]></javadoc>\n" +
-			"      <constructor jni-signature=\"(ILjava/lang/String;)V\">\n" +
-			"        <parameter jni-type=\"I\" name=\"one\" type=\"int\"/>\n" +
-			"        <parameter jni-type=\"Ljava/lang/String;\" name=\"two\" type=\"java.lang.String\"/>\n" +
-			"        <javadoc><![CDATA[jni-sig=<init>.(ILjava/lang/String;)V]]></javadoc>\n" +
-			"      </constructor>\n" +
-			"      <field jni-signature=\"I\" name=\"field\">\n" +
-			"        <javadoc><![CDATA[jni-sig=field.I]]></javadoc>\n" +
-			"      </field>\n" +
-			"      <method jni-return=\"V\" jni-signature=\"(Ljava/lang/Object;J)V\" name=\"m\" return=\"void\">\n" +
-			"        <parameter jni-type=\"Ljava/lang/Object;\" name=\"value\" type=\"T\"/>\n" +
-			"        <parameter jni-type=\"J\" name=\"x\" type=\"long\"/>\n" +
-			"        <javadoc><![CDATA[jni-sig=m.(Ljava/lang/Object;J)V]]></javadoc>\n" +
-			"      </method>\n" +
-			"    </class>\n" +
-			"    <interface jni-signature=\"LI;\" name=\"I\">\n" +
-			"      <javadoc><![CDATA[jni-sig=LI;]]></javadoc>\n" +
-			"      <method jni-return=\"Ljava/lang/Object;\" jni-signature=\"(Ljava/util/List;)Ljava/lang/Object;\" name=\"m\" return=\"T\">\n" +
-			"        <parameter jni-type=\"Ljava/util/List;\" name=\"x\" type=\"java.util.List&lt;T&gt;\"/>\n" +
-			"        <javadoc><![CDATA[jni-sig=m.(Ljava/util/List;)Ljava/lang/Object;]]></javadoc>\n" +
-			"      </method>\n" +
-			"    </interface>\n" +
-			"  </package>\n" +
-			"  <package jni-name=\"before/example\" name=\"before.example\"/>\n" +
-			"  <package jni-name=\"example\" name=\"example\">\n" +
-			"    <interface jni-signature=\"Lexample/Exampleable;\" name=\"Exampleable\">\n" +
-			"      <javadoc><![CDATA[jni-sig=Lexample/Exampleable;]]></javadoc>\n" +
-			"      <method jni-return=\"V\" jni-signature=\"(Ljava/lang/String;)V\" name=\"example\" return=\"void\">\n" +
-			"        <parameter jni-type=\"Ljava/lang/String;\" name=\"e\" type=\"java.lang.String\"/>\n" +
-			"        <javadoc><![CDATA[jni-sig=example.(Ljava/lang/String;)V]]></javadoc>\n" +
-			"      </method>\n" +
-			"      <method jni-return=\"V\" jni-signature=\"()V\" name=\"noParameters\" return=\"void\">\n" +
-			"        <javadoc><![CDATA[jni-sig=noParameters.()V]]></javadoc>\n" +
-			"      </method>\n" +
-			"    </interface>\n" +
-			"  </package>\n" +
-			"</api>\n";
+		final   String                  expected    = String.format(
```

Comment:
After sleeping on it, I wonder if using  [`String.replace()`](https://docs.oracle.com/javase/8/docs/api/java/lang/String.html#replace-java.lang.CharSequence-java.lang.CharSequence-) would make for a shorter `diff`:

```java
final string expected = 
    …
    .replace("\n", System.lineSeparator());
```

The entire string would thus be unchanged, and the diff would only show the removal of the last line (removing the `;`) and adding `.replace(…)`.

Reply:
Updated to reduce the overall diff.